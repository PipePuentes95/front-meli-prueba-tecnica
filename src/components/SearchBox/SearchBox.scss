@import '../../styles/variables';

.search-box {
  display: flex;
  align-items: center;
  justify-content: center;

  min-height: calc(100vh - 60px);

  h1 {
    font-size: 60px;
    text-align: center;
    color: $grey-3;

    @media (max-width: 576px) {
      font-size: 30px;
    }
  }
}
