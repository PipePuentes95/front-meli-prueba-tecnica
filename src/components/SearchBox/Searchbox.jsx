import React from 'react'

import './SearchBox.scss'

const SearchBox = () => {

  return (
    <div className='search-box'>
      <h1>Encuentra lo que estas buscando</h1>
    </div>
  );
};

export default SearchBox;
