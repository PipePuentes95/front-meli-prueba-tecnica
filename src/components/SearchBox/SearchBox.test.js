import { render } from '@testing-library/react';
import SearchBox from './SearchBox';

it('should take a snapshot', () => {
  const { asFragment } = render(<SearchBox />);

  expect(asFragment(<SearchBox />)).toMatchSnapshot();
});
