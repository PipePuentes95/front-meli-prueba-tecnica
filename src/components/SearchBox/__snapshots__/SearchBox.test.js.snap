// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="search-box"
  >
    <h1>
      Encuentra lo que estas buscando
    </h1>
  </div>
</DocumentFragment>
`;
